postgres/helm/postgres/values.yaml

```diff
@@ -8,9 +8,9 @@ operatorConfigName: operator-configuration
 oauthSecretName: postgres-operator
 
 image:
-  registry: registry.opensource.zalan.do
-  repository: acid/postgres-operator
-  tag: v1.6.3
+  registry: dkr.plural.sh
```

Comment:
could we use a gcr or ghcr repo for this?

Reply:
We could, but I thought we wanted to unify using the plural registry so people know where all images come from and that they have. been scanned for vulnerabilities. This would also be needed for vulnerability based upgrade policies.